src/relentless/simulate/initialize.py

```diff
@@ -123,11 +124,16 @@ def _pack_particles(seed, N, V, diameters):
         types = []
         trees = {}
         Nadded = 0
+        evaluated_diameters = copy.deepcopy(diameters)
```

Comment:
If I did not deepcopy, then when I evaluated the diameters they were being permanently evaluated. So when I returned the diameter variable, it was not longer an object. I'm not sure if there is a better way to handle this.

Reply:
Since `diameters` is a `dict`, it is a reference and will be modified if you update its contents in some way. See below for a compact way I think can achieve what we want without `copy`.